airbyte-integrations/connectors/source-monday/source_monday/spec.json

```diff
@@ -4,14 +4,81 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Monday Spec",
     "type": "object",
-    "required": ["api_token"],
-    "additionalProperties": false,
+    "required": [],
+    "additionalProperties": true,
     "properties": {
-      "api_token": {
-        "type": "string",
-        "description": "This is the API token to authenticate requests to Monday. Profile picture (bottom left) => Admin => API",
-        "airbyte_secret": true
+      "credentials": {
+        "title": "Authorization Method",
+        "type": "object",
+        "oneOf": [
```

Comment:
Actually in your case we don't need to have two kinds of credentials.

What to do for original spec.json file:
- Add non-required 'subdomain' property.
- Yout Aouth2 flow should return api_token (instead of access_token now)